Gemfile

```diff
@@ -20,6 +20,9 @@ end
 # but our runtime dep is still 3.9+
 gem "rspec", ">= 3.10"
 
+# checks code offenses with inspec check
+gem "cookstyle"
+
```

Comment:
This should probably be a gemspec dependency, since we want to be able to run inspec check from gem installs.